tests/lib/rules/yoda.js

```diff
@@ -134,23 +134,50 @@ ruleTester.run("yoda", rule, {
             options: ["never", { exceptRange: true }],
             parserOptions: { ecmaVersion: 2018 }
         }, {
-            code: "if (x <= `bar` || `foo` < x) {}",
+            code: "if ('a' < x && x < MAX ) {}",
             options: ["always", { exceptRange: true }],
             parserOptions: { ecmaVersion: 2015 }
         }, {
-            code: "if (`blue` < x.y && x.y < `green`) {}",
+            code: "if ('a' < x && x < MAX ) {}",
+            options: ["always"],
+            parserOptions: { ecmaVersion: 2015 }
+        }, {
+            code: "if (x <= 'foo' || 'bar' < x) {}",
+            options: ["always", { exceptRange: true }],
+            parserOptions: { ecmaVersion: 2015 }
+        }, {
+            code: "if (MIN < x && x < 'a' ) {}",
             options: ["never", { exceptRange: true }],
             parserOptions: { ecmaVersion: 2015 }
         }, {
+            code: "if (MIN < x && x < 'a' ) {}",
+            options: ["never"],
+            parserOptions: { ecmaVersion: 2015 }
+        },
+        {
+            code: "if (`blue` < x.y && x.y < `green`) {}",
+            options: ["never", { exceptRange: true }],
+            parserOptions: { ecmaVersion: 2015 }
+        },
+        {
+            code: "if ('a' < x && x < MAX ) {}",
+            options: ["never", { exceptRange: true }],
+            parserOptions: { ecmaVersion: 2015 }
```

Comment:
I think we don't need to specify the `parserOptions` in these cases (non-template literals).

Reply:
ops....forgot to remove this. 

i will change it 👍

Reply:
done